lib/getProcessStat.js

```diff
@@ -26,9 +35,9 @@ function getProcessStat(pid, callback)
        'cguest_time', 'start_data', 'end_data', 'start_brk', 'arg_start',
        'arg_end', 'env_start', 'env_end', 'exit_code')
 
-   callback(null, result)
+   return callback(null, result)
```

Comment:
Why the `return`? The function is not returning nothing at all, the fact of being used before is only just a convenience...